ui/src/app/views/rename-modal/component.tsx

```diff
@@ -87,26 +53,23 @@ export class RenameModal extends React.Component<RenameModalProps, {
         message: 'already exists'
       }
     }});
-    return <Modal loading={saving}>
-      <ModalTitle>Please enter the new name for {notebookName}</ModalTitle>
+    return <Modal>
+      <ModalTitle>Please enter new name for {oldName}</ModalTitle>
```

Comment:
Should this also have a description? Perhaps optionally? If so, can we also use this for the cohortRename modal? (I think its called `cohort-edit-modal`?)